charts/ctms/templates/secrets.yaml

```diff
@@ -27,9 +27,21 @@ metadata:
   name: {{ include "ctms.fullname" . }}
 data:
   # echo -n "postgres://ctms:defaultpassword@postgres:5432/ctms" | base64 -w 0
-  DATABASE_URL: cG9zdGdyZXM6Ly9tZW50b3Jpbmc6ZGVmYXVsdHBhc3N3b3JkQHBvc3RncmVzOjU0MzIvbWVudG9yaW5n
+  CTMS_DB_URL: cG9zdGdyZXM6Ly9jdG1zOmRlZmF1bHRwYXNzd29yZEBwb3N0Z3Jlczo1NDMyL2N0bXM=
   # echo -n "this-is-not-the-production-value" | base64
-  DJANGO_SECRET_KEY: dGhpcy1pcy1ub3QtdGhlLXByb2R1Y3Rpb24tdmFsdWU=
+  CTMS_SECRET_KEY: dGhpcy1pcy1ub3QtdGhlLXByb2R1Y3Rpb24tdmFsdWU=
+  CTMS_ACOUSTIC_LOOP_MIN_SECS: NQ==
+  # false
```

Comment:
I have questions, but they don't need to prevent merging, since this is clearly a known factor of some sort.